client/package.json

```diff
@@ -17,5 +17,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "proxy": "http://127.0.0.1:8080"
```

Comment:
There is a feature that comes with `create-react-app` that allow you to redirect unresolved traffic to a fallback server. This allows us to deal with [cors](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS) and simplify our calls.

It is possible to deal with this on the server side as well but this technique is less popular in projects where the frontend and the backend are separate entities.